acai-ios/AppDelegate.swift

```diff
@@ -36,7 +36,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func setupWindow() {
         window = UIWindow()
-        window?.rootViewController = UINavigationController(rootViewController: MenuViewController())
+        window?.rootViewController = UINavigationController(rootViewController: LoginViewController())
```

Comment:
discussed offline but should start thinking about how to setup so that a user doesn't have to relogin everytime. i'm thinking store the auth token in `UserDefaults` and check that to see if they need to relogin